frontend/app/src/i18n/en.json

```diff
@@ -616,5 +616,6 @@
     "websiteVersion": "of the website",
     "advanced": "Advanced",
     "nMessagesDeleted": "{number} message(s) deleted",
```

Comment:
You don't need the (s) in brackets anymore (same with the translations)

Reply:
More haste less speed!